core/src/main/c/share/vec_agg.cpp

```diff
@@ -167,29 +167,34 @@ int64_t SUM_LONG(int64_t *pl, int64_t count) {
 int64_t MIN_LONG(int64_t *pl, int64_t count) {
     Vec8q vec;
     const int step = 8;
-    Vec8q vecMin = L_MAX;
+    Vec8q vecMin = L_MIN;
```

Comment:
Have you considered the simpler null-count approach?
Something like:
```
min = L_MAX;
nullcount = if_add(bVec, nullcount, 1);
...
return (min == L_MAX && nullcount == count) ? L_MIN : min;
```

Reply:
Let me try that.